code/modules/projectiles/guns/projectile/revolver.dm

```diff
@@ -109,9 +109,9 @@
 		if(prob(70 - (magazine.ammo_count() * 10)))	//minimum probability of 10, maximum of 60
 			playsound(user, fire_sound, 50, 1)
 			to_chat(user, "<span class='userdanger'>[src] blows up in your face!</span>")
-			user.take_organ_damage(0,20)
+			user.take_organ_damage(60)
```

Comment:
github wont let me comment on it but please clean up the `as mob|obj|turf` and such in the parameters.